src/cli/README.md

```diff
@@ -479,7 +480,7 @@ Add a valid prefix to the Network Data.
 * prf: Default Router Preference, which may be: 'high', 'med', or 'low'.
 
 ```bash
-> route add 2001:dead:beef:cafe::/64 pvsr 0
+> route add 2001:dead:beef:cafe::/64 s med
```

Comment:
Notice that in CLI function ProcessRouteAdd(), if added route prefix is stable, will not process prf parameter.